Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.svc.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.6 AS builder
+FROM registry.svc.ci.openshift.org/ocp/builder:golang-1.15 AS builder
 WORKDIR /ingress-operator
 COPY . .
 RUN make build
```

Comment:
I am unsure if `Dockerfile` should use `registry.svc.ci.openshift.org/ocp/4.0:base` or `registry.svc.ci.openshift.org/ocp/4.6:base` in the next build stage (technically no diff in this file, but it would be a diff from `Dockerfile.rhel7`).